smac/initial_design/initial_design.py

```diff
@@ -98,6 +98,8 @@ def __init__(
 
     def select_configurations(self) -> typing.List[Configuration]:
         """Selects the initial configurations."""
+        if self.init_budget == 0:
```

Comment:
what is `init_budget`?

Reply:
The number of the initial budgets. This name comes from the raw SMAC implementation (see roughly 15 lines above) and currently I don't want to change that to other names as other functions might require this attribute